.github/workflows/wheels.yml

```diff
@@ -0,0 +1,33 @@
+name: Python wheel package build and publish
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  wheels:
+    runs-on: ${{ matrix.os }}
+    defaults:
+      run:
+        shell: bash -l {0}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu-latest"]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Build manylinux Python wheels
+      uses: RalfG/python-wheels-manylinux-build@v0.3.4-manylinux2014_x86_64
+      with:
+        python-versions: 'cp37-cp37m cp38-cp38 cp39-cp39'
+        build-requirements: 'cffi numpy cython'
```

Comment:
Can we be more explicit with the NumPy version?  We should build using NumPy 1.18.* as of today according to [NEP 29](https://numpy.org/neps/nep-0029-deprecation_policy.html).

Reply:
Done